oaknational/Oak-Web-Application
# chore: create sitemap.xml file

## Description

- Adds a sitemap.xml file to curriculum journey
- Updates sitemap-config file

## Issue(s)

Fixes #CUR-312

## How to test

1. Go to https://deploy-preview-2022--oak-web-application.netlify.thenational.academy/teachers/curriculum/sitemap.xml
2. You should see sitemap for all the possible curriculum phase options for units and overview pages

## Screenshots

How it used to look (delete if n/a):
<img width="1142" alt="Screenshot 2023-10-17 at 09 56 00" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/a3ae38c0-481c-4d39-92e9-8cbc86e8dcc1">


How it should now look:
<img width="1484" alt="Screenshot 2023-10-17 at 09 55 11" src="https://github.com/oaknational/Oak-Web-Application/assets/91190841/86335271-2824-45ac-8b17-34ba13c81390">


## Checklist
